### PR TITLE
Create 2018-08-26-thoughts-about-ai-bot-wechaty.md

### PR DESCRIPTION
**Pull request by lirobin1998**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.
e.g.
```
---
 layout: post
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```
- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [x] Add `<!--more-->` section to just show the abstract in blog homepage.

[See more](https://github.com/Chatie/blog#contribute-guideline)

### CODE REVIEW

**Review by lijiarui — CHANGES_REQUESTED** [review]
Thanks for your pull request.
1 . Please change the filename from `2018-08-26-Thoughts-About-AI-Bot-WeChaty` to `2018-08-26-thoughts-about-ai-bot-wechaty.md`
2. Please use markdown form instead of the raw text form.
3. See more guidline in [CONTRIBUTE GUIDELINE](https://github.com/chatie/blog#contribute-guideline)

File: _posts/2018-08-26-Thoughts-About-AI-Bot-WeChaty [hunk 1]
```diff
@@ -0,0 +1,267 @@
+# AI、Bot、WeChaty相关思考
```
Comment (lijiarui) [inline _posts/2018-08-26-Thoughts-About-AI-Bot-WeChaty:1]: Please add the header at the top of this blog:

```
---
 layout: post
 title: "AI、Bot、WeChaty相关思考"
 date: 2018-08-30 09:00 +0800
 author: lirobin1998
 ---
 
 > Author: [@李彬](https://github.com/lirobin1998) ，清华大学博士。硅谷智慧科技公司创始人，贵州博士科技有限公司CEO。聚焦智慧AI创新创业，从AI科普教育切入，开展人机智慧智能开发教育，打造全民AI教育开放平台。30年前在清华大学电子工程系开始研究机器翻译，20年前到硅谷开始学习探索智慧智能科技，10年硅谷与国内高科技创业经历。

```

File: _posts/2018-08-26-Thoughts-About-AI-Bot-WeChaty [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+---
+
+人工智能（AI：Artificial Intelligence）经历了六十年几起几落，终于在2016年因为阿尔法狗（AlphaGo）战胜围棋世界冠军而轰动全球，迎来了发展新机遇。2017年中国发布《新一代人工智能发展规划》，准备到2025年把中国建设成为全球最重要的AI创新中心之一。中国新一代AI的创新机会主要在弱AI的广泛应用和强AI的原始创新，而Bot（聊天机器人）是使用AI科技成果的一种最简洁形式。目前市场上具有“公众号+Bot+内容”这样的AI使用解决方案，支持大众以Bot形式使用AI科技成果。同时市场上也有像WeChaty这样的软件开源Bot项目，支持研发人员开发和集成各种各样的Bot引擎，支持Bot引擎的原始创新。显然，把AI、新一代AI、Bot、WeChaty这些概念和关系梳理清楚，有助于我们更好地推动中国AI的发展。
+
```
Comment (lijiarui) [inline _posts/2018-08-26-Thoughts-About-AI-Bot-WeChaty:5]: add `<!--more-->` here

File: _posts/2018-08-26-Thoughts-About-AI-Bot-WeChaty [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+首先，可以完善“微信号+WeChaty+Bot引擎”这个解决方案，提供SaaS服务模式为更多的用户服务。这样普通大众不需要做任何代码开发就可以使用WeChaty。
+
+其次，在“微信号+WeChaty+Bot引擎”这个解决方案的SaaS服务平台上不断添加新的功能供用户选择使用，如更多的微信社交功能、更多的Bot引擎等，使WeChaty的这套解决方案具有更强的功能、更多的用户、更高的价值。
+
+最后，通过“微信号+WeChaty+Bot引擎”这个解决方案的SaaS服务平台开发，使WeChaty这个软件开源项目更有价值，成为开放开源的AI创新创业平台。用户不仅可以在这个平台上开展AI教学、学习这个平台相关的AI教学内容，而且可以开发各种创新创业开源项目，特别是新一代人工智能的Bot引擎项目。
+
+## 五、作者简介
+---
+> 李彬，清华大学博士。硅谷智慧科技公司创始人，贵州博士科技有限公司CEO。聚焦智慧AI创新创业，从AI科普教育切入，开展人机智慧智能开发教育，打造全民AI教育开放平台。30年前在清华大学电子工程系开始研究机器翻译，20年前到硅谷开始学习探索智慧智能科技，10年硅谷与国内高科技创业经历。
```
Comment (lijiarui) [inline _posts/2018-08-26-Thoughts-About-AI-Bot-WeChaty:267]: move this to the top of the blog as I showed.

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request, and:

Change the filename from `2018-08-26-Thoughts-About-AI-Bot-WeChaty` to `2018-08-26-thoughts-about-ai-bot-wechaty.md`

**Comment by lirobin1998** [comment]
created a new pull request. I am closing this pull request.

**Pull request closed without merging**